lib/src/migrators/module.dart

```diff
@@ -284,83 +308,95 @@ class _ModuleMigrationVisitor extends MigrationVisitor {
-          span = span.file.span(span.start.offset + 1, span.end.offset - 1);
-        }
-        addPatch(Patch(span, name));
+
+      _patchNamespaceForFunction(node, (namespace) {
         var beforeParen = node.span.end.offset - 1;
-        if (namespace != null) {
-          addPatch(Patch(node.span.file.span(beforeParen, beforeParen),
-              ', \$module: "$namespace"'));
-        }
-      });
+        addPatch(Patch(node.span.file.span(beforeParen, beforeParen),
+            ', \$module: "$namespace"'));
+      }, getFunctionCall: true);
     }
   }
 
-  /// Calls [patcher] when the function [node] with name [originalName] requires
-  /// a namespace and/or a new name and adds a new use rule if necessary.
+  /// Calls [namespacePatcher] when the function [node] requires a namespace.
+  ///
+  /// This also patches the name for any built-in functions whose names change
+  /// in the module system.
   ///
   /// When the function is a color function that's not present in the module
   /// system (like `lighten`), this also migrates its `$amount` argument to the
   /// appropriate `color.adjust` argument.
   ///
-  /// [patcher] takes two arguments: the name used to refer to that function
-  /// when namespaced, and the namespace itself. The name will match the name
-  /// provided to the outer function except for built-in functions whose name
-  /// within a module differs from its original name.
-  void _patchNamespaceForFunction(FunctionExpression node, String originalName,
-      void patcher(String name, String namespace)) {
-    if (originalName == null) return;
-    if (_scope.isLocalFunction(originalName)) return;
-
-    var name = _unprefix(originalName) ?? originalName;
-
-    var namespace = _scope.global.functions.containsKey(name)
-        ? _namespaceForNode(_scope.global.functions[name])
-        : null;
-
-    if (namespace == null && builtInFunctionModules.containsKey(name)) {
-      // Don't migrate CSS-compatibility overloads.
-      if (_isCssCompatibilityOverload(node)) return;
-
-      namespace = builtInFunctionModules[name];
-      name = builtInFunctionNameChanges[name] ?? name;
-      if (namespace == 'color' && removedColorFunctions.containsKey(name)) {
-        if (node.arguments.positional.length == 2 &&
-            node.arguments.named.isEmpty) {
-          _patchRemovedColorFunction(name, node.arguments.positional.last);
-          name = 'adjust';
-        } else if (node.arguments.named.containsKey('amount')) {
-          var arg = node.arguments.named['amount'];
-          _patchRemovedColorFunction(name, arg,
-              existingArgName: _findArgNameSpan(arg));
-          name = 'adjust';
-        } else {
-          emitWarning("Could not migrate malformed '$name' call", node.span);
-          return;
-        }
+  /// If [node] is a get-function call, [getFunctionCall] should be true.
+  void _patchNamespaceForFunction(
+      FunctionExpression node, void namespacePatcher(String namespace),
```

Comment:
Functions, even callbacks, should usually be verb phrases (unless they're just `callback` or `fn` or something), so I'd call `namespacePatcher` something like `patchNamespace`.